src/main.rs

```diff
@@ -1,11 +1,15 @@
+#[cfg(not(tracermode = "hw"))]
```

Comment:
This seems like a weird check for `ykrustc`: what happens when we want to try this with the software tracer?

Reply:
Then you can add a `or` clause.

There's currently no other way of knowing if the compiler is ykrustc. We could add one, but it's not top priority right now.

Reply:
Should we just remove this check then?

Reply:
If we are happy with:
```
error[E0432]: unresolved import `ykrt`
```

instead.

Reply:
Yep!